samples/hellofs/hello_fs.go

```diff
@@ -218,7 +218,7 @@ func (fs *helloFS) ReadDir(
 
 	// Grab the range of interest.
 	if op.Offset > fuseops.DirOffset(len(entries)) {
-		return fuse.EIO
+		return nil
```

Comment:
Can you explain why it’s safe to change the error return in these samples? Was it wrong to begin with? Doesn’t the change affect the existing implementations?

Reply:
I'm pretty much sure the correct convention in FUSE is to return a nil response rather than an error. I guess that linux  FUSE implementation is a bit lenient in that case. I could do the same but it would require a special treatment of this specific use case which I'd rather not do. I think the better way is to return nil in the user code implementation